docs/querying/sql-data-types.md

```diff
@@ -74,14 +74,13 @@ The following table describes how Druid maps SQL types onto native types when ru
 
 ## Multi-value strings
 
-Druid's native type system allows strings to potentially have multiple values. These
-[multi-value string dimensions](multi-value-dimensions.md) are reported in SQL as VARCHAR typed, and can be
-syntactically used like any other VARCHAR. Regular string functions that refer to multi-value string dimensions are
-applied to all values for each row individually. Multi-value string dimensions can also be treated as arrays via special
-[multi-value string functions](sql-multivalue-string-functions.md), which can perform powerful array-aware operations, but retain
-their VARCHAR typing and behavior.
+Druid's native type system allows strings to have multiple values. These [multi-value string dimensions](multi-value-dimensions.md) are reported in SQL as type VARCHAR and can be
+syntactically used like any other VARCHAR. Regular string functions that refer to multi-value string dimensions are applied to all values for each row individually.
 
-Grouping by a multi-value expression observes the native Druid multi-value aggregation behavior, which is similar to the SQL UNNEST operator. Refer to the documentation on [multi-value string dimensions](multi-value-dimensions.md) for additional details.
+You can treat multi-value string dimensions as arrays using special
+[multi-value string functions](sql-multivalue-string-functions.md), which perform powerful array-aware operations, but retain their VARCHAR type and behavior.
+
+When grouping on a multi-value dimension, Druid uses all values from matching rows to generate one group per value, similar to using the UNNEST operator on an ARRAY type. See [Grouping](multi-value-dimensions.md#grouping) for more information.
```

Comment:
The similarity isn't that there's one group per value. UNNEST doesn't do any grouping. It's that when you do a group by for N MVDs, that can return more than N results because each MVD has multiple values. This is like when you unnest N arrays, you can have more than N results because each array is comprised of multiple values